internal/models/objecttype/object_type.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package objecttypemodel
+
+import (
+	"github.com/go-openapi/swag"
+)
+
+// VmwareTanzuCoreV1alpha1ObjectType Holds general type metadatas.
+//
+// swagger:model vmware.tanzu.core.v1alpha1.object.Type
+type VmwareTanzuCoreV1alpha1ObjectType struct {
```

Comment:
Do we need these values?

Reply:
We need this as the API response contains it.

Reply:
We need not populate these values as there no use of it. Very similar to creationTime or ResourceVersion in case of clusters

Reply:
Moved under policy, will be removing it after testing for CRUD ops.